### PR TITLE
feat: wishlist page and functionality

### PR DESCRIPTION
**Pull request by aatishpandey**
Wishlist Page and functionality.
image

### CODE REVIEW

File: src/components/WishlistCard/wishlist-card.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.card-horizontal {
+    max-width: 100%;
+    width: 100%;
+    margin-top: 0;
+}
+
+.cart-main {
+    display: grid;
+    grid-template-columns: 3fr 1fr;
```
Comment (rahulcs754) [inline src/components/WishlistCard/wishlist-card.css:9]: for design use grid that is good

File: src/components/WishlistCard/wishlist-card.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import "./wishlist-card.css"
+import { useCart } from "../../contexts/cartContext";
+import { Link } from "react-router-dom"
+
+const WishlistCard = ({ product }) => {
+    const { cartDispatch, cartState } = useCart();
+    const { myCart } = cartState;
+    const flag = myCart.find(item => item.id === product.id);
+    const gotoCardText = "Go to Cart"
+    const addToCartText = "Add to Cart"
```
Comment (rahulcs754) [inline src/components/WishlistCard/wishlist-card.js:10]: nice to use variable for text

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean .
i liked some thing in your.

File: src/pages/wishlistpage/wishlist-page.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { WishlistCard } from "../../components/WishlistCard/wishlist-card";
+import { useCart } from "../../contexts/cartContext"
+import "./wishlist-page.css"
+
+const WishPage = () => {
+    const { cartState } = useCart();
+    const { myWishlist, WishlistFlag } = cartState;
+    console.log(WishlistFlag)
```
Comment (coldpigli) [inline src/pages/wishlistpage/wishlist-page.jsx]: Remove the log

**Review by coldpigli — APPROVED** [review]
Small suggestion mentioned. Other than that I think everything looks good.

File: src/pages/wishlistpage/wishlist-page.jsx (hunk 3 above)
Comment (aatishpandey) [reply src/pages/wishlistpage/wishlist-page.jsx]: thanks for pointing out

**Pull request merged**